release-policies.md

```diff
@@ -1,52 +1,54 @@
+* **Standard Support** releases are supported for eighteen months, released in even-numbered years. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation.
+* **LTS** releases are supported for three years, released in odd-numbered years. They are intended for users that want the stability and lower cost of maintaining an application for an extended period, only needing to upgrade their .NET version for security patches.
 
-A single "bug bar" is used to decide if a change is warranted and safe for servicing updates. A given fix is often applied to multiple [servicing branches](https://github.com/dotnet/core/blob/main/daily-builds.md#servicing-releases), independent of release type. Breaking changes are not accepted during servicing, except (in the very rare case) to resolve a security vulnerability.
+Note: **Standard** releases were previously called **Current**.
 
-Improvements are released as "patches". Patch releases are cumulative. Patches are released on the Microsoft "Patch Tuesday" (second Tuesday of each month), however there is no guarantee that there will be a .NET release on any given Patch Tuesday. Patches are announced on the [.NET blog](https://devblogs.microsoft.com/dotnet/). A digest of monthly releases is published to [dotnet/announcements](https://github.com/dotnet/announcements/labels/Monthly-Update).
+LTS and Standard Support releases differ only by support duration. The .NET team follows the same software engineering and release processes for both release types, including for security, compatibility, and reliability. Both releases may contain major new features and breaking changes. The .NET team aspires to enable straightforward migration from one release to another, independent of release type.
 
-### Full support
+## Support phases
 
-During the full support period, .NET releases are updated to improve functional capabilities and mitigate security vulnerabilities.
+.NET releases go through multiple support phases, with varying support levels.
 
-Functional improvements are typically very targeted, and may address the following:
+* **Preview** releases are not supported but are offered for public testing and for the opportunity to give feedback.
+* **Release Candidate** releases have a go-live support that enables users to deploy a pre-release build in production and are supported by Microsoft.
+* **Active** support is provided for the majority of the period after a release is generally available (GA). Functional and security improvements will be provided, including support for new operating system versions.
+* **Maintenance** support is provided for the last six months of support. Improvements are limited to security fixes. Support for new operating system versions will be provided on a best-effort basis.
+* **End of support** or end-of-life (EOL) marks the point where a release is no longer supported.
```

Comment:
```suggestion
* **End of support** or end-of-life (EOL) marks the point where a release no longer receives servicing updates and is no longer supported.
```

Reply:
I don't see that this addition is necessary. It is fully implied and called out in full detail in a later section.